generate-types.sh

```diff
@@ -28,4 +28,6 @@ import { LRParser } from '@lezer/lr'
 
 export const parser: LRParser
 $(sed -E 's/ = [0-9]+/: number/' src/parser.terms.js)
+
+export const formatLokiQuery: (query: string) => string
```

Comment:
Do you know what this file does?

Reply:
yeah this generates the ts types that are exported with the package,
if we don't add to this, when importing `formatLokiQuery` into grafana we get the following error
`Module '"@grafana/lezer-logql"' has no exported member 'formatLokiQuery' .ts(2305)`

Reply:
👌